src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -76,24 +77,18 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		if (tableColumns) {
 			let mycolumns: Slick.Column<any>[] = [];
 			let index: number = 0;
+
 			(<any[]>columns).map(col => {
 				if (col.type && col.type === 1) {
 					this.createCheckBoxPlugin(col, index);
 				}
 				else if (col.type && col.type === 2) {
 					this.createButtonPlugin(col);
+				} else if (col.type && col.type === 3) {
```

Comment:
Let's use the actual ColumnType enum for these

Reply:
if I write col.type === azdata.ColumnType.icon the rintime throw error:
```
 "[uncaught exception]: Error: Cannot find module 'azdata'
Require stack:
- C:\ViaCode\azuredatastudio\out\vs\code\electron-browser\workbench\workbench.html", source: file:///C:/ViaCode/azuredatastudio/out/bootstrap-window.js (216)
[7704:1030/194901.456:INFO:CONSOLE(219)] "Error: Cannot find module 'azdata'
```